packages/react-native-editor/metro.config.js

```diff
@@ -18,5 +18,4 @@ module.exports = {
 			},
 		} ),
 	},
-	maxWorkers: 2,
```

Comment:
In the past, [we went from 3 to 2](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1980) to decrease the flakiness of the tests run. Now, I understand that we are now running the tests in a different infrastructure (GitHub Actions on macos this time) but, what do you think about postponing this change and do it in a separate PR? I'm concerned we might increase flakiness if we let all workers engage.

Unless of course this change is important for this PR and the primary fix. Thoughts?

Reply:
I think those maxWorkers are jest's maxWorkers and not metro's. The metro maxWorkers change was introduced in the monorepo branch (probably to reduce any flakiness), but seems like it's not necessary because CI ran fine 25 times without it as well and it could be beneficial to use more workers also when running the metro server locally.